pkgs/tools/cd-dvd/cdrkit/default.nix

```diff
@@ -44,10 +44,8 @@ stdenv.mkDerivation rec {
   '';
 
   postConfigure = lib.optionalString stdenv.hostPlatform.isDarwin ''
-    for f in */CMakeFiles/*.dir/link.txt ; do
-      substituteInPlace "$f" \
-        --replace "-lrt" "-framework IOKit"
-    done
+    substituteInPlace  */CMakeFiles/*.dir/link.txt \
+      --replace-warn "-lrt" "-framework IOKit -framework CoreFoundation"
```

Comment:
Should probably be `--replace-fail`? Although I doubt this package is ever going to change again.

Reply:
no. there are some files without matches. could be `replace-quiet`